src/main/webapp/app/config/constants/regex.spec.ts

```diff
@@ -47,4 +47,25 @@ describe('Regex constants test', () => {
       expect(mixText.split(REF_CAPTURE).length).toEqual(5);
     });
   });
+
+  describe('Valid latin text regex', () => {
```

Comment:
We can make use of test.each() https://jestjs.io/docs/api#each when the test logic is the same, just different parameters (test cases).